src/app/footer/footer.component.html

```diff
@@ -121,7 +121,7 @@ <h4 class="footer-widget-title" translate="footer.our_address">Our Address</h4>
             </div>
         </div>
         <div class="footer-bottom-row text-center">
-            <p class="app-credit small">&copy;2018 CTemplar. <span translate="footer.all_rights_reserved">All Rights Reserved</span>.</p>
+            <p class="app-credit small"> © 2018 Templar Software Systems Ltd., a Seychelles Limited Liability Company</p>
```

Comment:
Any specific reason to replace `&copy;`? I think its better to use `&copy;` for compatibility purpose.

Reply:
no specific reason, I will replace it.